src/server/server.js

```diff
@@ -47,8 +47,16 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(requestIp.mw());
 
+const checkAuthorizationHeader = (req, res, next) => {
+  const { authorization } = req.headers;
+  if (authorization !== `ApiKey ${process.env.SERVER_API_KEY}`) {
```

Comment:
Here it should be `Authorization` instead of `authorization`, and `config.SERVER_API_KEY` instead of `process.env.SERVER_API_KEY`, where `config` can be imported as `import config from 'config';`, cause this is server-side-only code.